src/recordlinker/linking/link.py

```diff
@@ -51,7 +51,7 @@ def compare(
     details: dict[str, typing.Any] = {"patient.reference_id": str(patient.reference_id)}
     for e in evals:
         # TODO: can we do this check earlier?
-        feature = getattr(schemas.Feature, e.feature, None)
+        feature = schemas.Feature.parse(e.feature)
```

Comment:
calling the parse method here means we have to evaluate the string every time we run a feature comparison, which can add up.  If we do it in say the `bound_evaluators` call, we only have to do it once per request.  I know that simple change would create a model->schema dependency which isn't ideal, but can you think of another way to do this earlier in the stack?